src/dsp_tools/commands/validate_data/models/input_problems.py

```diff
@@ -146,12 +147,15 @@ class InputProblem(ABC):
     prop_name: str
 
     @property
+    @abstractmethod
```

Comment:
If you use the decorator you force a child class to overwrite the function, which is what we want.